_includes/releases/v22.1/v22.1.0-beta.1.md

```diff
@@ -55,9 +52,9 @@ Release Date: April 4, 2022
 - Fixed a bug whereby certain catalog interactions which occurred concurrently with node failures were not internally retried. [#78698][#78698]
 - Fixed a bug that caused the optimizer to generate invalid query plans which could result in incorrect query results. The bug, which has been present since version v21.1.0, can appear if all of the following conditions are true:
   1. The query contains a semi-join, such as queries in the form: `SELECT * FROM t1 WHERE EXISTS (SELECT * FROM t2 WHERE t1.a = t2.a);`.
-  1. The inner table has an index containing the equality column, like `t2.a` in the example query.
-  1. The index contains one or more columns that prefix the equality column.
-  1. The prefix columns are `NOT NULL` and are constrained to a set of constant values via a `CHECK` constraint or an `IN` condition in the filter. [#78972][#78972]
+  2. The inner table has an index containing the equality column, like `t2.a` in the example query.
+  3. The index contains one or more columns that prefix the equality column.
+  4. The prefix columns are `NOT NULL` and are constrained to a set of constant values via a `CHECK` constraint or an `IN` condition in the filter. [#78972][#78972]
```

Comment:
All lists items should be numbered with `1. ` See https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#lists